python/pyiceberg/catalog/__init__.py

```diff
@@ -120,16 +120,16 @@ def load_catalog(name: str, **properties: Optional[str]) -> Catalog:
             or if it could not determine the catalog based on the properties
     """
     env = _ENV_CONFIG.get_catalog_config(name)
-    conf = merge_config(env or {}, properties)
+    conf = merge_config(env or {}, properties)  # type: ignore
```

Comment:
@LuigiCerone I noticed that you ignored the type-check here. Anything that we can do to make this pass instead of suppressing it?

Reply:
Yes @Fokko , I'll try to work on that! :)